src/utils.cpp

```diff
@@ -1335,4 +1337,49 @@ std::tuple<std::string, std::string, std::string> split_addrrange_symbol_module(
                          symbol.length() - idx2 - strlen(" []")) };
 }
 
+std::map<uint64_t, elf_symbol, std::greater<uint64_t>> get_symbol_table_for_elf(
```

Comment:
That is because the semantics of `std::map::lower_bound` and `std::map::upper_bound`, citing CPPreference:
- `std::map::lower_bound`: Returns an iterator pointing to the first element that is not less than (i.e. greater or equal to) key.
- `std::map::upper_bound`:  Returns an iterator pointing to the first element that is greater than key.

I want the element that is *less or equal* to the key (possible start address of the symbol), therefore I have to sort the elements of the map the other way.